### PR TITLE
dependabot eslint plugin related changes.

### PR DESCRIPTION
**Pull request by hsubramanianaks**
This PR supersedes https://github.com/Azure/vscode-aks-tools/pull/833 and fixes the eslint errors while running in ubuntu/linux.

Here is failing run - https://github.com/Azure/vscode-aks-tools/actions/runs/10256769131/job/28378026288?pr=833

passing run - https://github.com/Azure/vscode-aks-tools/actions/runs/10356661850/job/28666966196?pr=845

### CODE REVIEW

**Comment by ReinierCC** [comment]
Looks good, went through and reviewed for matching return types.

**Review by ReinierCC — APPROVED** [review]
Looks good, went through and reviewed for matching return types.

File: src/commands/draft/draftCommands.ts [hunk 1]
```diff
@@ -235,8 +235,8 @@ async function getRepo(octokit: Octokit, remote: Remote): Promise<GitHubRepo | n
 
     let response: RestEndpointMethodTypes["repos"]["get"]["response"];
     try {
-        response = await octokit.rest.repos.get({ owner, repo });
-    } catch (e) {
+        response = await octokit.repos.get({ owner, repo });
```
Comment (ReinierCC) [inline src/commands/draft/draftCommands.ts:238]: Think the change back to "octokit.repos.get" might be a mistake.

**Review by ReinierCC — CHANGES_REQUESTED** [review]

File: src/commands/draft/draftCommands.ts (hunk 1 above)
Comment (Tatsinnit) [reply src/commands/draft/draftCommands.ts:238]: lemme quickly fix this. nice!

**Review by ReinierCC — APPROVED** [review]

**Pull request merged**